tools/wptrunner/wptrunner/testrunner.py

```diff
@@ -229,7 +229,6 @@ def stop(self, force=False):
     def cleanup(self):
         if self.init_timer is not None:
             self.init_timer.cancel()
-        self.browser.cleanup()
```

Comment:
Wait was this removal intentional?

Reply:
Yes, indeed, and it's quite important. See the commit message:

> Also fix testrunner so that setup & cleanup balance out (called at enter
> & exit of a Browser context, respectively). Previously, cleaup() didn't
> do anything in all browsers, so the issue wasn't discovered. In this
> commit, cleanup() is used for uninstalling port forwarding.

which is all about the removal of this one line.